src/util.go

```diff
@@ -13,6 +13,20 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+// Illustrate example incoming YAML data structure for test purposes
+var ExampleYamlData = []byte(
+	`
+- name: Test
+  vars:
+    insights_signature: "ascii_armored gpg signature"
+    insights_signature_exclude: "/vars/insights_signature,/vars/content_vars"
+    content: |
+        #!/bin/sh
+        echo "$RHC_WORKER_FOO $RHC_WORKER_BAR!"
+    content_vars:
+        FOO: Hello
+        BAR: World`)
+
```

Comment:
Not sure if this should belong here. We could create a special folder for testing and place those fixtures there. I think it would make more sense other than placing any test specific example data in a module that is used during the build, rather than only be used during the tests.

Reply:
Yeah, you are totally right - I initially just wanted to try move it to another file to see if it works for both tests and then forgot about it :facepalm: 

I created `fixtures_test.go` for it